design/login.go

```diff
@@ -20,6 +20,7 @@ var _ = a.Resource("login", func() {
 				a.Enum("offline_access")
 				a.Description("If scope=offline_access then an offline token will be issued instead of a regular refresh token")
 			})
+			a.Param("api_token", d.String, "If not an empty string then return a token for accessing API")
```

Comment:
Could we call it `api_client` instead ?

Reply:
Yeah.. maybe api_client is better.. I don't like api_token much but couldn't come up with anything better ;)

Reply:
Renamed!